stock_picking_portal/models/res_config_settings.py

```
@@ -12,26 +12,16 @@ class ResConfigSettings(models.TransientModel):
-        )
-        if portal_visible_operation_ids:
-            res.update(
-                portal_visible_operation_ids=[
-                    int(r) for r in portal_visible_operation_ids.split(",")
-                ]
-            )
+        visible = self.env["stock.picking.type"].search([("portal_visible", "=", True)])
+        res.update(portal_visible_operation_ids=visible.ids)
+        return res
+
+    def set_values(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `set_values`

(ANN201)